frontend/src/app/features/team-planner/team-planner/planner/team-planner.component.html

```diff
@@ -95,25 +95,26 @@
     </ng-template>
 
     <ng-template #eventContent let-event="event">
-  
-      <op-wp-loading-skeleton 
+
+      <op-wp-loading-skeleton
         [viewBox]="event.extendedProps.viewBox"
         class="op-team-planner--wp-loading-skeleton"
         *ngIf="event.source && event.source.id === 'skeleton'"
       ></op-wp-loading-skeleton>
 
       <wp-single-card
-        *ngIf="!event.source || event.source.id === 'work_packages'"
-        [workPackage]="event.extendedProps.workPackage"
+        *ngIf="(!event.source || event.source.id === 'work_packages') && event.extendedProps.workPackage as wp"
+        [workPackage]="wp"
         [selectedWhenOpen]="true"
         [orientation]="'horizontal'"
         [highlightingMode]="'type'"
         [showInfoButton]="true"
-        [disabledInfo]="showDisabledText(event.extendedProps.workPackage)"
-        [isClosed]="isStatusClosed(event.extendedProps.workPackage)"
+        [disabledInfo]="showDisabledText(wp)"
+        [isClosed]="isStatusClosed(wp)"
+        [showAsGhost]="shouldShowAsGhost(wp.id, (globalDraggingItem$ | async))"
         [showAsInlineCard]="true"
-        [showStartDate]="!this.isWpDateInCurrentView(event.extendedProps.workPackage, 'start')"
-        [showEndDate]="!this.isWpDateInCurrentView(event.extendedProps.workPackage, 'end')"
+        [showStartDate]="!this.isWpDateInCurrentView(wp, 'start')"
+        [showEndDate]="!this.isWpDateInCurrentView(wp, 'end')"
```

Comment:
Not a huge issue, but FYI this'll cause a merge conflict here https://github.com/opf/openproject/pull/10516/files#diff-3c1aafc75343b7ef47e93b0d524120031e7f2fb52384aa94aeaa16cd0c7dda6dR115